hwdb/60-keyboard.hwdb

```diff
@@ -979,6 +986,12 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*550P*:pvr*
  KEYBOARD_KEY_a8=!                                      # Fn Lock - Function lock on
  KEYBOARD_KEY_a9=!                                      # Fn Lock - Function lock off
 
+# Ativ 6
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*670Z*:pvr*
+ KEYBOARD_KEY_ce=!prog1                                 # Fn+F1 launch settings
+ KEYBOARD_KEY_97=!kbdillumdown                          # Fn+F9 keyboard backlight down
+ KEYBOARD_KEY_96=!kbdillumup                            # Fn+F10 keyboard backlight up
+
```

Comment:
This looks like the existing entry for `# Series 7 / 9` can you just add the new ID there? Or are some of the assigned keys wrong there?